IS7024FinalProjectBreweryArc/BreweryArc/Pages/Index.cshtml

```diff
@@ -100,7 +102,7 @@
             </tr>
             @{
                 
-                    <tr>
+                    <tr style="background-color:ButtonHighlight">

```

Comment:
Inline styles are not permitted in XHTML 1.0 Strict.  Inline styles cause bloat, and tie content to presentation.  Use linked styles instead.